### PR TITLE
feat(event_type): Added event_type to event_info return

### PR DESCRIPTION
**Pull request by scott-glover020**
This is the back end changes that need to be made in order to get the event type.

### CODE REVIEW

File: api/app/events/tests.py [hunk 1]
```diff
@@ -209,6 +209,16 @@ def test_get_events_unauthed_not_available(self):
         assert data[0]["description"] == 'Event Description'
         assert data[0]["status"] == 'Application not available'
 
+    def test_get_event_type(self):
+        self.seed_static_data()
+        db.session.delete(self.test_form)
+        db.session.commit()
+
+        response = self.app.get('/api/v1/events')
+        assert response.status_code == 200
```
Comment (avishkar58) [inline api/app/events/tests.py]: Please use `self.assertEqual(response.status_code, 200)` instead - this provides better error messages when the assertion fails (I know the rest of the file does it with Python assert, but we're slowly trying to change this :) )

Comment (avishkar58) [reply api/app/events/tests.py]: Bonus points if you'd like to clean up the rest of the asserts 🥇

Comment (scott-glover020) [reply api/app/events/tests.py]: Ye Ill give it a go!

**Comment by scott-glover020** [comment]
Refactored the assert cases to self.assertEqual for the events tests

**Comment by steveskimmel** [comment]
Added a commit for the front end change to support this backend update

**Pull request merged**